.env.example

```
@@ -27,3 +27,10 @@ NEXT_PUBLIC_ENABLE_NOTIFICATIONS=
 
 NEXT_PUBLIC_AUTH_PROFILE_TIMEOUT_MS=4000 # timeout (ms) before forcing incomplete profile redirect
 NEXT_PUBLIC_QURAN_REFLECT_URL=
+
+# Playwright tests
+DISCORD_BOT_TOKEN=
+DISCORD_CHANNEL_ID=
+TEST_USER_EMAIL=
+TEST_USER_PASSWORD=
+TEST_USER_USERNAME=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)